src/components/repl.tsx

```diff
@@ -82,27 +82,71 @@ const Repl: ReplProps = (props) => {
   }
   const [edit, setEdit] = createSignal(-1);
   const [outputTab, setOutputTab] = createSignal(0);
-
-  let model: editor.ITextModel;
+  let outputModel: editor.ITextModel;
+  let importMapModel: editor.ITextModel;
+  const [importMap, setImportMap] = createSignal<any>({}, { equals: false });
```

Comment:
When you share a link, this doesn't seem to be saved. Is it possible to make sure this gets included in the exported files (and then imported back)